app/generators/number_generator.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class NumberGenerator < QuestionTypeGenerator
```

Comment:
kun je hierin niet gewoon de textfield generator aanroepen met de goeie regex?

Reply:
Ja vaak wel, en dat is ook de reden dat we dit component niet gebruiken in de evaluatievragenlijst. Maar soms wil je gewoon een numberinput met arrow up en down controls waar het letterlijk onmogelijk is om iets behalve cijfers in te voeren en waar je gewoon van kan zetten ik wil een nummer groter dan 589 en kleiner dan 829 en dat is wat moeiljk te doen met regex dus daar is deze control voor.

Maar inderdaad op dit moment wordt ie niet gebruikt.